gce.go

```diff
@@ -45,7 +45,8 @@ const (
 
 // GCEReqInstanceDiscovery work unit for a pool of GCEDiscovery workers
 type GCEReqInstanceDiscovery struct {
-	Project           string
+	Project string
```

Comment:
in `func NewGCEDiscovery()` you create a new context. Accept it as a parameter and pass it from `NewGCEDiscoveryPool`.

Reply:
Maybe even add the ctx to the `GCEDiscovery` struct